spec/lib/miq_preloader_spec.rb

```diff
@@ -100,6 +100,17 @@
       expect { expect(nodes.first.container_images).to eq([image]) }.to make_database_queries(:count => 1)
     end
 
+    
+    it "preloads association array" do
+      host = FactoryBot.create(:host_vmware) # infra
+      FactoryBot.create_list(:vm, 2, :ext_management_system => FactoryBot.create(:ems_infra), :host => host)
+      emses = ExtManagementSystem.all
+      expect { preload(emses, [:vms, :host]) }.to make_database_queries(:count => 2)
```

Comment:
hosts? plural?

```
```
  1) MiqPreloader.preload preloads association array
     Failure/Error: ActiveRecord::Associations::Preloader.new(:records => Array(records), :associations => associations, :scope => preload_scope).call

     ActiveRecord::AssociationNotFoundError:
       Association named 'host' was not found on ManageIQ::Providers::Vmware::InfraManager; perhaps you misspelled it?
     # ./lib/miq_preloader.rb:25:in `preload'
     # ./spec/lib/miq_preloader_spec.rb:115:in `preload'
     # ./spec/lib/miq_preloader_spec.rb:108:in `block (4 levels) in <top (required)>'
     # ./spec/lib/miq_preloader_spec.rb:108:in `block (3 levels) in <top (required)>'
```

Reply:
ugh. thnx